### DISCUSSION TITLE
Have problem with thumbnail generator

### DISCUSSION THREAD

**Discussion by GrzesMis** [mriscoc/Ender3V2S1#1335] [General]
Hello anybody knows why I have ths error? Cant export Gcode, need to delete file location from post processing scripts
![Zrzut ekranu 2024-08-01 201142](https://github.com/user-attachments/assets/8dc5a0af-850f-4829-a800-c8b56ea8c13c)

**Comment by mriscoc** [comment]
Try with shorter paths, there is a troubleshooting guide: https://github.com/mriscoc/Ender3V2S1/wiki/How-to-generate-a-gcode-preview#troubleshooting

**Reply by GrzesMis** [reply]
Cant make shorter paths there is nothing about my error in this guide
